stl/modules/std.compat.ixx

```diff
@@ -0,0 +1,532 @@
+export using std::ilogb;
+export using std::lgamma;
+export using std::log1p;
+export using std::log2;
+export using std::logb;
+export using std::llrint;
+export using std::lrint;
+export using std::nearbyint;
+export using std::rint;
+export using std::llround;
+export using std::lround;
+export using std::fdim;
+export using std::fma;
+export using std::fmax;
+export using std::fmin;
+export using std::round;
+export using std::trunc;
+export using std::remainder;
+export using std::remquo;
+export using std::copysign;
+export using std::nan;
+export using std::nextafter;
+export using std::scalbn;
+export using std::scalbln;
+export using std::nexttoward;
+export using std::tgamma;
+export using std::acoshf;
+export using std::asinhf;
+export using std::atanhf;
+export using std::cbrtf;
+export using std::erff;
+export using std::erfcf;
+export using std::expm1f;
+export using std::exp2f;
+export using std::hypotf;
+export using std::ilogbf;
+export using std::lgammaf;
+export using std::log1pf;
+export using std::log2f;
+export using std::logbf;
+export using std::llrintf;
+export using std::lrintf;
+export using std::nearbyintf;
+export using std::rintf;
+export using std::llroundf;
+export using std::lroundf;
+export using std::fdimf;
+export using std::fmaf;
+export using std::fmaxf;
+export using std::fminf;
+export using std::roundf;
+export using std::truncf;
+export using std::remainderf;
+export using std::remquof;
+export using std::copysignf;
+export using std::nanf;
+export using std::nextafterf;
+export using std::scalbnf;
+export using std::scalblnf;
+export using std::nexttowardf;
+export using std::tgammaf;
+export using std::acoshl;
+export using std::asinhl;
+export using std::atanhl;
+export using std::cbrtl;
+export using std::erfl;
+export using std::erfcl;
+export using std::expm1l;
+export using std::exp2l;
+export using std::hypotl;
+export using std::ilogbl;
+export using std::lgammal;
+export using std::log1pl;
+export using std::log2l;
+export using std::logbl;
+export using std::llrintl;
+export using std::lrintl;
+export using std::nearbyintl;
+export using std::rintl;
+export using std::llroundl;
+export using std::lroundl;
+export using std::fdiml;
+export using std::fmal;
+export using std::fmaxl;
+export using std::fminl;
+export using std::roundl;
+export using std::truncl;
+export using std::remainderl;
+export using std::remquol;
+export using std::copysignl;
+export using std::nanl;
+export using std::nextafterl;
+export using std::scalbnl;
+export using std::scalblnl;
+export using std::nexttowardl;
+export using std::tgammal;
+export using std::fpclassify;
+export using std::signbit;
+export using std::isfinite;
+export using std::isinf;
+export using std::isnan;
+export using std::isnormal;
+export using std::isgreater;
+export using std::isgreaterequal;
+export using std::isless;
+export using std::islessequal;
+export using std::islessgreater;
+export using std::isunordered;
+

```

Comment:
No change requested.
Currently [[support.c.headers.other]/1](https://eel.is/c++draft/support.c.headers.other#1) requires provide `::lerp` in `<math.h>`, and thus `std.compat` is also required to provide it. I guess that this is an oversight in [P0811R3](https://wg21.link/p0811r3), and I've tried to file an LWG issue to add `lerp` to the excluded list.

Reply:
Ditto "`<cmath>` declares `std::lerp` so `std.compat` is required to export it in the global namespace per [std.modules]/3".

Reply:
Thanks for filing LWG-3782. As mentioned in my other reply, #3111 is the followup issue for `std.compat`.